### DISCUSSION TITLE
creating a .env file

### DISCUSSION THREAD

**Discussion by essnuked** [Cyfrin/foundry-full-course-cu#2276] [General]
I am creating the .env file beneath the test folder but in the video, i can see its on its own.

how do i create the .env file so it stands on its own?

i created the file under the test folder but when i run " source  .env "

it gives me this error message " no such file or directory"

**Comment by EngrPips** [comment]
Hello @essnuked, Please show your working directory as I show mine below.

<img width="1440" alt="Screenshot 2024-08-20 at 08 28 18" src="https://github.com/user-attachments/assets/878727ac-44f6-494d-b985-723b6c2df111">

**Reply by essnuked** [reply]
I MANAGED TO FIGURE IT OUT, THANK YOU!

**Reply by EngrPips** [reply]
Awesome, Keep crushing it.
